Samples/WebSignIn/Common (iOS)/Keychain/Keychain+Token.swift

```diff
@@ -22,17 +22,23 @@ public extension Keychain {
     private static let oktaMobileSdkAccessGroup = "com.okta.mobile-sdk.shared"
     
     static func saveDeviceSSO(_ token: Token) throws {
-        try [Token.Kind.idToken, Token.Kind.deviceSecret].forEach { kind in
-            try token.token(of: kind)
-                .flatMap { $0.data(using: .utf8) }
-                .map {
-                    Keychain.Item(account: "Okta-\(kind.rawValue)",
-                                  service: "Okta",
-                                  accessGroup: oktaMobileSdkAccessGroup,
-                                  value: $0)
-                }
-                .map{ try $0.save() }
+        try [Token.Kind.idToken, Token.Kind.deviceSecret]
+            .forEach { try saveDeviceSSO(token, kind: $0) }
+    }
+    
+    static func saveDeviceSSO(_ token: Token, kind: Token.Kind) throws {
```

Comment:
Curious - can we save multiple tokens of the same `Kind`? Am assuming that our `account` value below is our distinguishing characteristic

Reply:
You'll want to check out this other PR that extended my Keychain implementation for more information https://github.com/okta/okta-mobile-swift/pull/39

Ultimately this class is very flexible and clean, so it should be useful for you. 

This use-case in the sample app is only there to save the device secret and ID token with an accompanying sample app to demonstrate Native SSO.